datajoint/base_relation.py

```diff
@@ -177,7 +177,7 @@ def make_placeholder(name, value):
                     value = pack(value)
                     placeholder = '%s'
                 elif heading[name].numeric:
-                    if value is None or np.isnan(np.float(value)):  # nans are turned into NULLs
+                    if value in [None, ''] or np.isnan(np.float(value)): # nans are turned into NULLs
```

Comment:
actually, I am not sure.  It's just a feeling I had.  Something about always using `is None` rather than hashing `None`.